### PR TITLE
fix(*): Updates for JSX compatibility

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: SURF-1219 (part 1), SURF-879

Preview: https://deploy-preview-289--helix-ui.netlify.com/

### LGTM's
- [x] Dev LGTM
- [x] Info Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

**Comment by CITguy** [comment]
Need to add FAQ for which APIs are covered by webcomponents/webcomponentsjs and `helix-ui.browser[.min].js`.

File: docs/docs.less [hunk 1]
```diff
@@ -264,3 +264,19 @@ dl:not(.hxList) {
...
+
+iframe[src*="codepen.io"] {
+  border: 2px solid @gray-900;
+  height: 25rem;
+  width: 100%;
+}
+
+hx-icon {
+  &.native-support {
+    color: @green-500;;
```
Comment (nimbinatus) [inline docs/docs.less]: extra semicolon

File: docs/guides/getting-started/index.html [hunk 2]
```diff
@@ -57,66 +59,167 @@ <h2 id="install">Step 1: Install HelixUI</h2>
...
+            <a href="https://unpkg.com">unpkg.com</a> via the following URLs.
+          </p>
+
+          <h3>Development</h3>
+          <ul>
+            <li><code>https://unpkg.com/helix-ui@{{VERSION}}/dist/styles/helix-ui.css</code></li>
+            <li><code>https://unpkg.com/helix-ui@{{VERSION}}/dist/scripts/helix-ui.browser.js</code></li>
+          </ul>
+
+          <h3>Production (minified):</h3>
```
Comment (nimbinatus) [inline docs/guides/getting-started/index.html]: Remove colon at end.

File: docs/guides/getting-started/index.html [hunk 3]
```diff
@@ -57,66 +59,167 @@ <h2 id="install">Step 1: Install HelixUI</h2>
...
     <ol>
       <li>
-        Copy the source code for a layout template into your application.
+        Install <code>@webcomponents/webcomponentsjs</code> via one of the methods shown below.
       </li>
       <li>
-        Review the copied markup (especially URLs) to fit your application's needs.
+        Copy or extract the polyfills into your application's target output directory.
       </li>
+      <li>Proceed to "Step 3: Choose a Layout"</li>
```
Comment (nimbinatus) [inline docs/guides/getting-started/index.html]: Add a period at the end.

File: docs/guides/getting-started/index.html [hunk 4]
```diff
@@ -57,66 +59,167 @@ <h2 id="install">Step 1: Install HelixUI</h2>
...
+            Using <a href="https://npmjs.com">npm</a>:
           </p>
-        </div>
-        <footer>
-          <a href="components/layouts/vertical-layout-template.html">
-            View the Vertical Template
-          </a>
-        </footer>
-      </hx-panel>
+          <figure>
```
Comment (nimbinatus) [inline docs/guides/getting-started/index.html]: figure not needed? Unwrap.

File: docs/guides/getting-started/index.html [hunk 5]
```diff
@@ -57,66 +59,167 @@ <h2 id="install">Step 1: Install HelixUI</h2>
...
 
-      <hx-panel class="beta-hxCard hxOpen">
-        <header>
-          <h3>Horizontal Layout</h3>
-        </header>
-        <div>
+          <p>
+            If you prefer <a href="https://yarnpkg.com/">Yarn</a>, use the following command:
+          </p>
+          <figure>
```
Comment (nimbinatus) [inline docs/guides/getting-started/index.html]: figure not needed? Unwrap.

File: docs/guides/getting-started/index.html [hunk 6]
```diff
@@ -29,6 +30,7 @@ <h2 id="install">Step 1: Install HelixUI</h2>
               npm install helix-ui
             {% endcode %}
           </figure>
```
Comment (nimbinatus) [inline docs/guides/getting-started/index.html]: figure not needed? Unwrap.

File: docs/guides/getting-started/index.html [hunk 7]
```diff
@@ -57,66 +59,167 @@ <h2 id="install">Step 1: Install HelixUI</h2>
...
+        Install <code>@webcomponents/webcomponentsjs</code> via one of the methods shown below.
       </li>
       <li>
-        Review the copied markup (especially URLs) to fit your application's needs.
+        Copy or extract the polyfills into your application's target output directory.
       </li>
+      <li>Proceed to "Step 3: Choose a Layout"</li>
     </ol>
-    <div class="beta-hxCardRow">
-      <hx-panel class="beta-hxCard hxOpen">
```
Comment (nimbinatus) [inline docs/guides/getting-started/index.html]: Check where `beta-hxCard` is defined. Remove it or keep it where it is?

File: docs/guides/getting-started/index.html [hunk 8]
```diff
@@ -57,66 +59,167 @@ <h2 id="install">Step 1: Install HelixUI</h2>
...
+            <a
+              class="hxBtn hxPrimary"
+              download="HelixUI-Horizontal-index.html"
+              href="components/layouts/horizontal-layout-template.html"
+            >
+              <hx-icon type="download"></hx-icon>
+              Download Template
+            </a>
+            &nbsp;
+            <a target="_blank" href="components/layouts/horizontal-layout-template.html">
```
Comment (nimbinatus) [inline docs/guides/getting-started/index.html]: rels needed

File: docs/guides/getting-started/index.html [hunk 9]
```diff
@@ -57,66 +59,167 @@ <h2 id="install">Step 1: Install HelixUI</h2>
...
+            <a
+              class="hxBtn hxPrimary"
+              download="HelixUI-Vertical-index.html"
+              href="components/layouts/vertical-layout-template.html"
+            >
+              <hx-icon type="download"></hx-icon>
+              Download Template
+            </a>
+            &nbsp;
+            <a target="_blank" href="components/layouts/vertical-layout-template.html">
```
Comment (nimbinatus) [inline docs/guides/getting-started/index.html]: rels needed

File: docs/guides/react-compatibility/index.html [hunk 10]
```diff
@@ -0,0 +1,53 @@
...
+    </p>
+
+    <br />
+
+    <h3>Problem</h3>
+    <p class="comfortable">
+      JSX has been found to remove significant white space in certain scenarios
+      (see demo below).
+    </p>
+    <figure>
```
Comment (nimbinatus) [inline docs/guides/react-compatibility/index.html]: hxFigure after other one merges

File: docs/guides/polyfills/index.html [hunk 11]
```diff
@@ -0,0 +1,80 @@
...
+  </section>
+
+  <section>
+    <h2 id="additional-polyfills">Additional Polyfills</h2>
+    <p>
+      If your application requires polyfills beyond those listed above, they will
+      need to be added separately.
+    </p>
+    <p>
+      Below are some resources where you might find polyfills.
```
Comment (chri2547) [inline docs/guides/polyfills/index.html]: Access the following resources to find polyfills.

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Built with commit af4b3a39f6b6b37d4422b3cfcb33729de052888f

https://app.netlify.com/sites/helix-ui/deploys/5b61eae2fdd72a665a539f02

**Pull request merged**